### PR TITLE
Fixes

### PR DESCRIPTION
**Pull request by janvanmansum**
NO JIRA ISSUE, but basically the tool wasn't building yet and couldn't handle empty lines in the input.
#### When applied it will
- Fix the build
- Make sure empty lines are ignored and don't cause the program to terminate.

### CODE REVIEW

File: src/main/scala/nl/knaw/dans/easy/umd/InputRecord.scala [hunk 1]
```diff
@@ -16,9 +31,9 @@ object InputRecord {
   def parse(file: File): Try[Stream[InputRecord]] = Try {
     CSVParser.parse(file, Charsets.UTF_8, CSVFormat.RFC4180)
       .asScala
-      .filter(_.asScala.nonEmpty)
-      .drop(1)
+      .drop(1) // Ingore the header
```
Comment (janvanmansum) [inline src/main/scala/nl/knaw/dans/easy/umd/InputRecord.scala:34]: The `filter`-didn't do anything. I assume it was intended as the `withFilter`-call below.

File: src/test/scala/nl/knaw/dans/easy/umd/CommandLineOptionsSpec.scala [hunk 2]
```diff
@@ -20,7 +20,7 @@ import java.io.{ByteArrayOutputStream, File}
 import nl.knaw.dans.easy.umd.CustomMatchers._
 import org.scalatest.{FlatSpec, Matchers}
 
-class ComandLineOptionsSpec extends FlatSpec with Matchers {
+class CommandLineOptionsSpec extends FlatSpec with Matchers {
```
Comment (janvanmansum) [inline src/test/scala/nl/knaw/dans/easy/umd/CommandLineOptionsSpec.scala:23]: Typo

File: src/test/scala/nl/knaw/dans/easy/umd/CommandSpec.scala [hunk 3]
```diff
@@ -1,67 +0,0 @@
-/**
```
Comment (janvanmansum) [inline src/test/scala/nl/knaw/dans/easy/umd/CommandSpec.scala:1]: Removed this, as it will only be run if `deasy` is online.

**Comment by janvanmansum** [comment]
@DANS-KNAW/easy for review.

File: src/main/scala/nl/knaw/dans/easy/umd/InputRecord.scala [hunk 4]
```diff
@@ -16,9 +31,9 @@ object InputRecord {
...
       .toStream
+      .withFilter(_.asScala.size > 1) // Ignore empty or incomplete lines
```
Comment (rvanheest) [inline src/main/scala/nl/knaw/dans/easy/umd/InputRecord.scala:36]: Shouldn't this be `_.asScala.size >= 1`, which is equal to `_.asScala.size > 0`, which is equal to `_.asScala.nonEmpty`, which is equal to `!_.isEmpty`?

Comment (janvanmansum) [reply src/main/scala/nl/knaw/dans/easy/umd/InputRecord.scala:36]: When I tested it with `nonEmpty`, empty lines were _not_ ignored.

Comment (rvanheest) [reply src/main/scala/nl/knaw/dans/easy/umd/InputRecord.scala:36]: Mmm, that's surprising! Most likely an empty line will produce a `CSVRecord` with 1 element which is an empty `String`...

**Pull request merged**